modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -343,21 +357,13 @@ class K8sTaskHandler extends TaskHandler {
 
     @Override
     boolean checkIfCompleted() {
-        if( !podName ) throw new IllegalStateException("Missing K8s pod name - cannot check if complete")
+        if( !podName ) throw new IllegalStateException("Missing K8s $this.resourceType name - cannot check if complete")
         def state = getState()
         if( state && state.terminated ) {
-            if( state.nodeTermination instanceof NodeTerminationException ) {
-                // kee track of the node termination error
-                task.error = (NodeTerminationException) state.nodeTermination
-                // mark the task as ABORTED since thr failure is caused by a node failure
-                task.aborted = true
-            }
-            else {
-                // finalize the task
-                task.exitStatus = readExitFile()
-                task.stdout = outputFile
-                task.stderr = errorFile
-            }
+            // finalize the task
+            task.exitStatus = readExitFile()
+            task.stdout = outputFile
+            task.stderr = errorFile
             status = TaskStatus.COMPLETED
```

Comment:
This should go into a separate PR

Reply:
this one is not mine, perhaps wrong merge from master? no problem to find and revert

Reply:
so what about this? Shall I search how this change got in?

Reply:
ok, I'll do. thanks!